.ci/linux/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+#
+# This script runs the dotnet build without the sample projects
+#
+set -euxo pipefail
+
+# Remove sample projects
```

Comment:
nit: could we change this to "Remove Full Framework projects"? - there are some other changes with the same wording. 
That'd be more correct, e.g. 1 of those is a real agent project, not a sample, and with that wording probably we won't forget to exclude other Full Framework projects in case we add more.

Reply:
Sure
```suggestion
# Remove Full Framework projects
```